components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/OpcNodeModel.cs

```diff
@@ -59,20 +63,22 @@ public class OpcNodeModel {
         [DataMember(EmitDefaultValue = false, IsRequired = false)]
         public string DisplayName { get; set; }
 
-        /// <summary> Heartbeat </summary>
+        /// <summary> Heartbeat interval in milliseconds </summary>
         [DataMember(EmitDefaultValue = false, IsRequired = false)]
-        public int? HeartbeatInterval {
-            get => HeartbeatIntervalTimespan.HasValue ?
-                (int)HeartbeatIntervalTimespan.Value.TotalSeconds : default;
-            set => HeartbeatIntervalTimespan = value.HasValue ?
-                TimeSpan.FromSeconds(value.Value) : default;
-        }
+        public int? HeartbeatInterval { get; set; }
 
         /// <summary>
-        /// Heartbeat interval as TimeSpan.
+        /// HeartbeatInterval as TimeSpan.
         /// </summary>
-        [DataMember(EmitDefaultValue = false, IsRequired = false)]
-        public TimeSpan? HeartbeatIntervalTimespan { get; set; }
+        [IgnoreDataMember]
```

Comment:
>IgnoreDataMember [](http://example.com/codeflow?start=9&length=16)

this was by intention the stored value so we can host milliseconds as well. the HeartBeat was supposed to be ignore-able, somehow it slipped.

Reply:
BTW, `OpcPublishingIntervalTimespan` and `OpcSamplingIntervalTimespan` having `[IgnoreDataMember]` set means that those fields will not be included in serialization and deserialization. So even if somebody sets `OpcPublishingIntervalTimespan` in published nodes file we will not be reading it. Is that what we want here ?

Similarly, if we set `[IgnoreDataMember]` for `HeartBeat`, then that value will also not be read from published nodes file. So we will have issues with reading old published nodes files. Which is also not desirable, right ?

Reply:
I will revert changes to heartbeat for now.

Reply:
Reverted.